inst/apphelp/descriptionCnaTab.md

```diff
@@ -0,0 +1,26 @@
+**Load a valid copy number data file.** This file is used in cBioPortal for the copy number data panel (see image on the right.)
+
+---
+
+#### A minimal copy number data file with only the columns required for cBioPortal:
+
+The discrete copy number data file contain values that would be derived from copy-number analysis algorithms like GISTIC 2.0 or RAE. For each gene (row) in the data file, the following columns are required in the order specified:
+
+- **Hugo_Symbol (Required)**: A HUGO gene symbol.
```

Comment:
It is optional, if there is an Entrez_Gene_Id column - better write as follows:

**One or both of:**
- **Hugo_Symbol (Required)**: ...
- **Entrez_Gene_Id (Optional, but recommended)**: ...

_(This behaviour should be correctly displayed in the implementation of your code.)_